1.1/spec/11-Part-08.adoc

```diff
@@ -984,7 +984,7 @@ This function returns the minimum bounding box - a rectangle - of `geom`. Calcul
 ==== Function: geof:geometryN
 
 ```
-geof:geometryN (geom: ogc:geomLiteral): xsd:integer
+geof:geometryN (geom: ogc:geomLiteral, geomindex: xsd:integer): xsd:integer
```

Comment:
Shouldn't this be `geof:geometryN (geom: ogc:geomLiteral, geomindex: xsd:integer): geom: ogc:geomLiteral` where ` :geom: ogc:geomLiteral` is the return value of a _geom_?